lib/webidl2.js

```diff
@@ -198,7 +222,7 @@
 
     let last_token = null;
 
-    function consume(type, value) {
+    function consume_single(type) {
```

Comment:
Seems messy to have the `consume_single` and `consume(...candidates)`, `consume(...candidates)` should consume single without needing to call out to the consume_single (i.e., merge `consume_single()` into `consume()`'s for loop).